Add favicons for all devices and browsers

Arhell opened the pull request:
resolve https://github.com/laminas/laminas.github.io/issues/175

froschdesign commented:
@Arhell 
Thank you very much for your efforts, but unfortunately we have the [same problems again as in the past](https://github.com/laminas/laminas.github.io/pull/105):

- the favicons should be added to the [website repository](https://github.com/laminas/getlaminas.org)
- [each sub-project has its own logo](https://github.com/laminas/getlaminas.org/tree/master/public/images/logo) and should also have its own favicon
- the current favicon [does not work on dark backgrounds](https://github.com/laminas/laminas.github.io/pull/105#pullrequestreview-481174975)

Arhell commented:
@froschdesign  You're right, I already managed to forget that I also wanted to do this

Updated the icons, added a logo with a white background for the dark theme, but in general the site does not support the dark theme

Thanks

froschdesign commented:
> Updated the icons, added a logo with a white background for the dark theme, but in general the site does not support the dark theme

The dark theme of the website is not meant here, but the dark theme of a browser.

Please also see my other notes.
Thanks in advance! 👍🏻

Arhell commented:
Of course, I will do it in the next PRs
If these icons are suitable, is it possible to merge?

froschdesign commented:
> If these icons are suitable to this repository, is it possible to merge?

Unfortunately no, because no logo or image is to be added to this repository. The central place is: https://github.com/laminas/getlaminas.org/tree/master/public/images/favicon / https://getlaminas.org/images/favicon/…

Arhell commented:
but as I looked at this PR https://github.com/laminas/laminas.github.io/pull/159 refer specifically to this repository and not to the project as a whole, if I close this PR and add icons here https:// github.com/laminas/getlaminas.org/tree/master/public/images/favicon then they are not added


Here is an example
https://github.com/laminas/laminas.github.io/blob/master/index.html#L9
![Screenshot_1](https://user-images.githubusercontent.com/26163841/226837729-31ab4ded-7a9b-4c3e-9dfa-e4eb2bf2ef30.png)

Arhell commented:
or do you mean add the logos to the central repository and add links from there here?

Arhell commented:
> > …or do you mean add the logos to the central repository and add links from there here?
> 
> Correct!

Yeah, thanks, then I won't close for now, update the icons there and update the links here.
But there is a question, site.webmanifest may not work in a cross repository, may leave icons for chrome here?

froschdesign commented:
> But there is a question, site.webmanifest may not work in a cross repository, may leave icons for chrome here?

No icons here and I think the manifest file can be omitted because we are not producing a "web application".
The icons for the sub-projects alone are good and a necessary improvement.

froschdesign commented on components/index.html:
```diff
@@ -6,7 +6,16 @@
     <meta name="description" content="Overview of the Laminas Components packages.">
-    <link rel="shortcut icon" href="/img/trademark-laminas-foundation-rgb.svg">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="https://getlaminas.org/images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://getlaminas.org/images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://getlaminas.org/images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="https://getlaminas.org/images/favicon/site.webmanifest">
+    <link rel="mask-icon" href="https://getlaminas.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://getlaminas.org/images/favicon/favicon.ico">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="msapplication-config" content="https://getlaminas.org/images/favicon/browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
```
The sub-project "Components" has a different logo, so the current favicon is wrong here. See also:

- https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1478986847
- https://github.com/laminas/getlaminas.org/pull/136#issuecomment-1488540683

Arhell replied:
Done

froschdesign commented on mvc/index.html:
```diff
@@ -5,7 +5,17 @@
     <meta name="description" content="Overview of the Laminas MVC packages.">
-    <link rel="shortcut icon" href="/img/trademark-laminas-foundation-rgb.svg">
+
+    <link rel="apple-touch-icon" sizes="180x180" href="https://getlaminas.org/images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://getlaminas.org/images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://getlaminas.org/images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="https://getlaminas.org/images/favicon/site.webmanifest">
+    <link rel="mask-icon" href="https://getlaminas.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="https://getlaminas.org/images/favicon/favicon.ico">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="msapplication-config" content="https://getlaminas.org/images/favicon/browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
```
The sub-project "MVC" has a different logo, so the current favicon is wrong here. See also:

- https://github.com/laminas/laminas.github.io/pull/176#issuecomment-1478986847
- https://github.com/laminas/getlaminas.org/pull/136#issuecomment-1488540683

Arhell replied:
Done

froschdesign reviewed (changes requested):
Some more favicons are needed, as each sub-project has its own logo and should also have its own favicon.

weierophinney commented:
@froschdesign Is this ready to merge at this point? I feel I should defer to you, as I'm not entirely clear on what's going on with the favicon set...

froschdesign commented:
The XML cannot be accessed, but this is a website problem: https://github.com/laminas/getlaminas.org/blob/35903a8465c826cc814e5773d0fb942a386d0fe7/.platform.app.yaml#L82-L85

froschdesign commented:
@Arhell 
Thank you for your time, your patience and this contribution! 👍
